R/vdiagram-data.R

```diff
@@ -55,6 +67,10 @@ parseVDiagramData <- function(reportObject){
     minStage=minStage))
 }
 
+#' History Measurements Label
+#' @description A function that checks to see if there are any historic measurements and, if so, creates a label
```

Comment:
when looking in vdiagram-styles, it seems like the historic points are black, but the lines are blue.